pkg/build/builder/docker.go

```diff
@@ -413,6 +419,33 @@ func traverseAST(cmd string, node *parser.Node) int {
 	return index
 }
 
+// insertEnvAfterFrom inserts an ENV Dockerfile instruction right after the
+// FROM instruction of the given dockerfile to define the environment variables
+// from env and returns the new Dockerfile.
+func insertEnvAfterFrom(env []kapi.EnvVar, dockerfile []byte) []byte {
+	if len(env) == 0 {
+		// Just return a copy.
+		return append([]byte(nil), dockerfile...)
+	}
+	fromLine := dockerfileFromLineRegexp.Find(dockerfile)
```

Comment:
Which is wrong, because you can have multiple `FROM` lines in Dockerfile and the `docker build` will pickup the last one...

```console
docker build -t test .
Sending build context to Docker daemon 30.29 MB
Sending build context to Docker daemon 
Step 0 : FROM scratch
 ---> 
Step 1 : FROM centos:centos7
 ---> 7322fbe74aa5
Successfully built 7322fbe74aa5
```